opm/input/eclipse/EclipseState/Co2StoreConfig.hpp

```diff
@@ -67,7 +81,15 @@ class Deck;
 
     SaltMixingType string2enumSalt(const std::string& input) const;
     LiquidMixingType string2enumLiquid(const std::string& input) const;
-    GasMixingType string2enumGas(const std::string& input) const; 
+    GasMixingType string2enumGas(const std::string& input) const;
+    
+    std::map<std::string, int> cnames;
+    std::vector<EzrokhiTable> denaqa_tables;
+    std::vector<EzrokhiTable> viscaqa_tables;
+    double salt {0.0};
+    double MmNaCl = 58.44e-3;
+    double MmH2O = 18e-3;
```

Comment:
Static for the Mm since they dont change?

Reply:
Added `static constexpr` to both `Mm`-variables